ufo/static/ufoDropdownMenu.html

```diff
@@ -95,8 +98,7 @@
     <paper-dialog id="removeAdminDialog" with-backdrop>
       <paper-button on-tap="flipToDropdownFlowFromRemoveAdmin" class="header">
         <iron-icon icon="arrow-back" item-icon></iron-icon>
-      </paper-button>
-      <p>{{resources.removeAdminInstructions}}</p>
+      </paper-button> {{resources.removeAdminInstructions}}
```

Comment:
Sorry about that! I was experimenting with using it inside the button which requires the space after an icon and then the text all on the same line so I didn't change it back here. Done now!